test/Serialization/basic_sil.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: %empty-directory(%t)
-// RUN: %target-build-swift -Xfrontend -assume-parsing-unqualified-ownership-sil -emit-module -Xfrontend -disable-diagnostic-passes -force-single-frontend-invocation -o %t/def_basic.swiftmodule %S/Inputs/def_basic.sil
+// RUN: %target-build-swift -Xfrontend -assume-parsing-unqualified-ownership-sil -emit-module -Xfrontend -disable-diagnostic-passes -force-single-frontend-invocation -Xfrontend -enable-objc-interop -o %t/def_basic.swiftmodule %S/Inputs/def_basic.sil
```

Comment:
Your previous change makes me wonder if this would be better as `%target-swift-frontend` instead.  I think that cleaning up uses of `%target-build-swift` in favor of `%target-swift-frontend` would be a nice follow up.

Reply:
I'd need to look at this test again. I was trying to be conservative about switching from "build" tests to "less than build".

Reply:
Oh, yeah, I agree that being conservative is nice - hence the "follow up" :-).

Reply:
I looked at it again. I don't confident enough about this test to simplify it.